.github/workflows/release.yml

```diff
@@ -12,17 +12,17 @@ jobs:
       - name: Prepare repository
         run: git fetch --unshallow --tags
 
-      - name: Use Node.js 14.x
+      - name: Use Node.js 16.x
         uses: actions/setup-node@v1
```

Comment:
github actions should be updated to latest version (v3) to work correctly with node 16, current codes gives the following warning: 
Node.js 12 actions are deprecated. Please update the following actions to use Node.js 16: actions/checkout@v2, actions/setup-node@v1. For more information see: https://github.blog/changelog/2022-09-22-github-actions-all-actions-will-begin-running-on-node16-instead-of-node12/.
both setup node and checkout v3 are available